bazel/repositories.bzl

```diff
@@ -82,7 +82,18 @@ def proxy_wasm_cpp_host_repositories():
         patch_args = ["-p1"],
     )
 
-    # Core.
+    # Core deps. Try to keep up with Envoy versions:
+    # https://github.com/envoyproxy/envoy/blob/main/bazel/repository_locations.bzl
+
+    # Note: latest LTS release, ahead of Envoy to pick up a bugfix found in local fuzzing:
+    # https://github.com/abseil/abseil-cpp/commit/e7858c73279d81cbc005d9c76a385ab535520635
+    maybe(
+        http_archive,
+        name = "com_google_absl",
+        sha256 = "733726b8c3a6d39a4120d7e45ea8b41a434cdacde401cba500f14236c49b39dc",
+        strip_prefix = "abseil-cpp-20240116.2",
+        urls = ["https://github.com/abseil/abseil-cpp/archive/20240116.2.tar.gz"],
+    )
```

Comment:
A few things:
1) Again, this is unrelated to Envoy.
2) C++ Host does not rely on abseil, so it's weird adding it as a dependency here.
3) I believe this should be fixed by the update of `rules_fuzzing` to a newer version (which ships with abseil v20240116.1).

Reply:
> update of rules_fuzzing to a newer version

Thanks for identifying the Abseil dependency path. If only it were this simple. The next rules_fuzzing major version (v0.4) adds some dynamically loaded pydeps that I can't figure out (they want to be loaded via WORKSPACE...), and the latest version (v0.5) also bumps to Bazel 7.

Can we do the tactical (Abseil) fix while we sort out larger and more intrusive upgrades?

Reply:
If updating `rules_fuzzing` is problematic, then I guess we have to override abseil dependency here.

Could you change/update the comment to reflect that this is temporary override and that it should be removed once we update `rules_fuzzing`? Otherwise, it's going to stay here as an explicit dependency forever.

Perhaps a file an issue for that as well?